layers/+lang/ess/README.org

```diff
@@ -3,57 +3,63 @@
 [[file:img/r.jpg]]
 
 * Table of Contents                                         :TOC_4_org:noexport:
- - [[#install][Install]]
- - [[#options][Options]]
-   - [[#style][Style]]
-   - [[#roxygen][Roxygen]]
- - [[#key-bindings][Key Bindings]]
-   - [[#inferior-ess-processes][Inferior ESS Processes]]
-   - [[#evaluation-commands][Evaluation Commands]]
-   - [[#Developer][Developer]]
-   - [[#debugging][Debugging]]
-   - [[#help][Help]]
+ - [[Install][Install]]
```

Comment:
Aren’t these links supposed to start with `#`? And if you change them to use `#` (and without initial capital letters) instead, you will get a smaller diff (reduced to your additions only).

I just ran `./core/tools/spacefmt/spacefmt all` while on your branch, which indeed changed these lines to `#`. So you should probably use `#`.